quupe-RN/js/redux/modules/Token.js

```diff
@@ -0,0 +1,50 @@
+import { createToken, queryToken } from '../../config/models';
```

Comment:
we need a DELETE_TOKEN as well. If a user logs out, we need to delete that token. Please make a corresponding action, action creator, and a reducer to DELETE_TOKEN.